rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -978,6 +1009,25 @@ bool np_handler::reply_set_roomdata_internal(u32 req_id, std::vector<u8>& /*repl
 	return true;
 }
 
+bool np_handler::reply_set_roommemberdata_internal(u32 req_id, std::vector<u8>& /*reply_data*/)
+{
+	if (pending_requests.count(req_id) == 0)
```

Comment:
c++20 has a `contains` function for this exact use case

Reply:
it's copy pasted but alright ill change the rest too